stable/presto/templates/configmap-catalog.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Values.server.catalog.data }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "presto.catalog" . }}
+  labels:
+    app: {{ template "presto.name" . }}
+    chart: {{ template "presto.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  {{ .Values.server.catalog.data }}
+---
```

Comment:
I0429 16:29:02.853] [ERROR] templates/configmap-catalog.yaml: unable to parse YAML
I0429 16:29:02.853] 	error converting YAML to JSON: yaml: line 13: did not find expected key

Reply:
I've fixed it in below commits. Thanks for point it out